site/content/en/docs/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Kueue"
+linkTitle: "Documentation"
+weight: 20
+menu:
+  main:
+    weight: 20
+---
```

Comment:
Should we list the three main sections here like https://github.com/kubernetes-sigs/kueue/blob/main/docs/README.md